### PR TITLE
feat: add 404 page

### PR DESCRIPTION
**Pull request by valerydluski**
image
image

### CODE REVIEW

File: client/src/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+function Page(props: IProps) {
+  return (
+    <PageLayout loading={false} githubId={props.session.githubId}>
+      <Row justify="center" style={{ margin: '65px 0 25px 0' }}>
+        <Image src="/static/svg/err.svg" alt="Error 404" width={175} height={175} />
+      </Row>
+      <Row justify="center">
+        <h2 style={{ fontSize: '102px', marginBottom: 0 }}>404</h2>
```
Comment (apalchys) [inline client/src/pages/404.tsx:17]: Maybe h1?

```suggestion
        <h1 style={{ fontSize: '102px', marginBottom: 0 }}>404</h1>
```

File: client/src/pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      </Row>
+      <Row justify="center">
+        <h3>That page doesn't exist!</h3>
```
Comment (apalchys) [inline client/src/pages/404.tsx:20]: h2? remove exclamation? re-phrase?

```suggestion
        <h2>Sorry, Page Not Found</h2>
```

File: client/src/pages/404.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      </Row>
+      <Row justify="center">
+        <p>Sorry, the page you were looking for could not be found.</p>
+      </Row>
```
Comment (apalchys) [inline client/src/pages/404.tsx:24]: Maybe remove it? We already said that it's not found ?

**Comment by valerydluski** [comment]
@apalchys, thank you, I fixed your remarks 
at the moment it looks like this

File: client/src/pages/404.tsx (hunk 1 above)
Comment (valerydluski) [reply client/src/pages/404.tsx]: Done

**Review by mikhama — APPROVED** [review]

File: client/src/pages/404.tsx (hunk 2 above)
Comment (valerydluski) [reply client/src/pages/404.tsx]: Done

**Pull request merged**